billing/forms.py

```diff
@@ -93,3 +99,58 @@ def save(self, commit=True):
         if commit:
             m.save()
         return m
+
+
+class UpdatePurchaseOrderForm(ModelForm):
+    ba_id = forms.IntegerField(widget=forms.HiddenInput())
+
+    class Meta:
+        model = BillingAccount
+        fields = ["business_purchase_order"]
+        labels = {"business_purchase_order": "Purchase order"}
+        help_texts = {
+            "business_purchase_order": "If you have a purchase order it will be displayed on your invoices, you may change this at any time."
```

Comment:
```suggestion
            "business_purchase_order": "If you enter a purchase order number here it will be displayed on your invoices. You can change this at any time."
```